<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseNetworkStudent</class>
 <widget class="QWidget" name="CourseNetworkStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程网(学生版)</string>
  </property>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;课程列表&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="course_info_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>451</width>
     <height>192</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>课程检索</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>课程编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IdEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>课程名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>课程教师</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="teacherEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>课程类别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeBox">
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectedButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查看已选课列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unselectedButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查看未选课列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生成已选课表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
